### PR TITLE
:lock: Add UUID Dat Support

### PR DESCRIPTION
**Pull request by haynescd**
Add download token button:

image

### CODE REVIEW

File: src/pages/staticPages/webAPI/WebAPIPage.tsx [hunk 1]
```diff
@@ -27,6 +27,8 @@ export class UserDataAccessToken {
     }
 }
 
+const supported_dat_methods = ['oauth2', 'uuid'];
```
Comment (alisman) [inline src/pages/staticPages/webAPI/WebAPIPage.tsx:30]: we usually try to put these kinds of constants in all caps

Comment (alisman) [reply src/pages/staticPages/webAPI/WebAPIPage.tsx:30]: SUPPORTED_DAT_METHODS

File: src/shared/components/dataAccessTokens/DataAccessTokensDropdown.tsx [hunk 2]
```diff
@@ -49,6 +49,8 @@ export class DataAccessTokensDropdown extends React.Component<
     }
 
     @computed get getDatDropdownList(): any {
+        const supported_dat_methods = ['oauth2', 'uuid'];
```
Comment (alisman) [inline src/shared/components/dataAccessTokens/DataAccessTokensDropdown.tsx:52]: maybe put this in src/shared/constants.ts to avoid the duplication

Comment (haynescd) [reply src/shared/components/dataAccessTokens/DataAccessTokensDropdown.tsx]: of course! ahaha just updated

**Pull request merged**